### PR TITLE
Propagate correlation id from gallery to search service

### PR DESCRIPTION
**Pull request by skofman1**

### CODE REVIEW

**Comment by skofman1** [comment]
@maartenba, @xavierdecoster ,  is this ok to submit?

File: src/NuGetGallery/Controllers/ODataV2FeedController.cs [hunk 1]
```diff
@@ -169,8 +174,8 @@ public IHttpActionResult GetPropertyFromPackages(string propertyName, string id,
         {
             switch (propertyName.ToLowerInvariant())
             {
-                case "id": return Ok(id);
-                case "version": return Ok(version);
+            case "id": return Ok(id);
```
Comment (maartenba) [inline src/NuGetGallery/Controllers/ODataV2FeedController.cs]: Whitespace seems gone?

File: src/NuGetGallery/Controllers/ODataV2FeedController.cs [hunk 2]
```diff
@@ -377,5 +384,13 @@ public IHttpActionResult GetPropertyFromPackages(string propertyName, string id,
             }
             return updates;
         }
+
+        private void SetCorrelation()
+        {
+            if (_searchService is ICorrelated)
```
Comment (maartenba) [inline src/NuGetGallery/Controllers/ODataV2FeedController.cs]: Can we do this in the constructor instead? That way we don't have to think of calling this method explicitly.

**Comment by maartenba** [comment]
Minor comment (on adding SetCorrelation() to the constructor). Other than that looks good to me, :shipit: :)

File: src/NuGetGallery/Controllers/ODataV2FeedController.cs (hunk 2 above)
Comment (skofman1) [inline src/NuGetGallery/Controllers/ODataV2FeedController.cs]: Can't... the Request is null in the ctor, so I have no correlation id to pass along

Comment (maartenba) [inline src/NuGetGallery/Controllers/ODataV2FeedController.cs]: Right! Overlooked that one :S

**Pull request merged**